docs/user-guides/desktop/index.md

```diff
@@ -123,6 +119,10 @@ Before you can use Coder Desktop, you will need to sign in.
 
 1. Coder Connect is now running!
 
+## Troubleshooting
+
+- Do not install more than one copy of Coder Desktop. To avoid system VPN configuration conflicts, only one copy of `Coder Desktop.app` should exist on your Mac, and it must remain in `/Applications`.
```

Comment:
```suggestion
Do not install more than one copy of Coder Desktop. To avoid system VPN configuration conflicts, only one copy of `Coder Desktop.app` should exist on your Mac, and it must remain in `/Applications`.
```

If it's just one list item, let's let it stand alone for now, even if it makes for a weird-looking troubleshooting section